<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_plot_show</class>
 <widget class="QWidget" name="Form_plot_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(177, 177, 177);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>160</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
